doc/userguide/rules/file-keywords.rst

```diff
@@ -127,6 +127,17 @@ here: https://redmine.openinfosecfoundation.org/issues/437
 
 ``file.magic`` supports multiple buffer matching, see :doc:`multi-buffer-matching`.
 
+file.mimetype
+-------------
+
+Sticky buffer that matches on the MIME type guessed from the binary content of a file.
```

Comment:
Why should we use that over file.magic ?
(I can guess but we may want to document it, right ? )